cmake/public/LoadHIP.cmake

```diff
@@ -122,6 +122,14 @@ else()
   set(ROCTRACER_PATH $ENV{ROCTRACER_PATH})
 endif()
 
+# MAGMA_HOME
+if(NOT DEFINED ENV{MAGMA_HOME})
+  set(MAGMA_HOME ${ROCM_PATH}/magma)
+  set(ENV{MAGMA_HOME} ${ROCM_PATH}/magma)
```

Comment:
Why this is needed?
```suggestion
```

Reply:
Because `cmake/Modules/FindMAGMA.cmake` uses
```
FIND_LIBRARY(MAGMA_LIBRARIES magma
  HINTS $ENV{MAGMA_HOME} /usr/local/magma
  PATH_SUFFIXES lib)

FIND_PATH(MAGMA_INCLUDE_DIR magma.h
  HINTS $ENV{MAGMA_HOME} /usr/local/magma
  PATH_SUFFIXES include)
```